packages/lesswrong/components/common/ContentItemBody.tsx

```diff
@@ -261,11 +261,11 @@ class ContentItemBody extends Component<ContentItemBodyProps,ContentItemBodyStat
       const linkTags = this.htmlCollectionToArray(this.bodyRef.current.getElementsByTagName("a"));
       for (let linkTag of linkTags) {
         const tagContentsHTML = linkTag.innerHTML;
-        const href = linkTag.getAttribute("href");
+        const href = linkTag.getAttribute("href") ?? '';
         if (linkIsExcludedFromPreview(href))
           continue;
-        const id = linkTag.getAttribute("id");
-        const rel = linkTag.getAttribute("rel")
+        const id = linkTag.getAttribute("id") ?? '';
```

Comment:
Should be `undefined` rather than `''` I think (and same for others in this function)